File: android/gemcore/src/main/kotlin/com/gemwallet/android/domains/fiat/FiatConfig.kt
package com.gemwallet.android.domains.fiat

import com.gemwallet.android.domains.gemConfig

object FiatConfig {
    private val config by lazy { gemConfig.getFiatConfig() }

    val insufficientNetworkFeeBuyAmount: Int get() = config.insufficientNetworkFeeBuyAmount
}

File: ios/Packages/GemstonePrimitives/Sources/FiatConfig.swift
// Copyright (c). Gem Wallet. All rights reserved.

import Foundation
import class Gemstone.Config

public struct FiatConfig {
    private init() {}

    public static var insufficientNetworkFeeBuyAmount: Int {
        Int(Config.shared.getFiatConfig().insufficientNetworkFeeBuyAmount)
    }
}
